### ISSUE TITLE
Allow multiple selectors

### ISSUE THREAD

**Issue by stuartcusackie** [visuellverstehen/statamic-classify#24]
I was trying to target multiple elements like so:
`'p, ul, ol, table' => 'mb-4 last:mb-0 empty:h-8',`

But it produces the error:
`Expected selector, but <delimiter ">" at 10> found`

Nice package by the way. Would you believe I made something very similar: https://github.com/stuartcusackie/cdom

My package uses the PHP Simple HTML Dom package, which allows all kinds of CSS targeting:
https://simplehtmldom.sourceforge.io/docs/1.9/index.html#select-elements-using-css-selectors

It would be great if your package could handle any kind of CSS selectors.

**Comment by flemming-petersen** [comment]
Do I get it right, you want to add there props (mb-4 last:mb-0 empty:h-8) to each element?
Or should this (p, ul, ol, table) already be a chained selector?

**Comment by stuartcusackie** [comment]
Yes it would be to each element. If you wrote:

```
p, ul, ol, table { 
  margin-bottom: 1px; 
} 
```

then it would apply to each element separately.

**Comment by flemming-petersen** [comment]
You can that by simply writing the following:
```
    'default' => [
        'p' => 'mb-4 last:mb-0 empty:h-8',
        'ul' => 'mb-4 last:mb-0 empty:h-8',
        'ol' => 'mb-4 last:mb-0 empty:h-8',
        'table' => 'mb-4 last:mb-0 empty:h-8',
    ],
```

But I will agree by chaining them it would be cleaner.
On the other hand the following could be a bit messy
```
    'default' => [
        'p, ul, ol, table' => 'mb-4 last:mb-0 empty:h-8',
        'table' => 'text-blue-900',
    ],
```
What do you think?

**Comment by stuartcusackie** [comment]
I think that allowing [every kind of CSS chaining](https://www.w3schools.com/cssref/css_selectors.asp) is useful, and would make sense to anyone who works with CSS or JS. I would have thought that the **symfony/css-selector** package would have all of these selectors and chaining methods built in, but perhaps not.

On the other hand, it isn't much effort for me to declare a line for each element individually. It might get more difficult to manage if you are styling many elements, but it might be fine too.

**Comment by flemming-petersen** [comment]
As you mentioned it would be a nice feature. We will try to implement this in the following days/weeks.

**Comment by simonolog** [comment]
@stuartcusackie _"Good things come to those who wait"_ ... or something – anyways, we've finally implemented selector chaining in 19b33f06ecca441c2a8f7f68cbbe6915bc61f382 😄
